Samples/Unity/PlayFabLeaderboardsUnity/Assets/Scripts/SampleManager.cs

```diff
@@ -1,142 +1,140 @@
-﻿using PlayFab;
-using PlayFab.ClientModels;
-using System.Collections;
-using System.Collections.Generic;
-using UnityEngine;
-using UnityEngine.UI;
-
-public class SampleManager : MonoBehaviour
-{
-	public InputField inputScoreObject;
-
-	private Dictionary<string, int> GameStats = new Dictionary<string, int>();
-
-	void Start()
-	{
-		if (PlayfabManager.IsLoggedIn == false)
-		{
-			PlayfabManager.Login(OnLoginSuccess, OnLoginFailed);
-		}
-	}
-
-	private void OnGotLeaderboards()
-	{
-		GoToLeaderboards();
-	}
-	private void OnGotLeaderboardsFailed()
-	{
-		Debug.LogError("ERROR: Failed to retrieve Leaderboards. Verify configuration on PlayFab GameManager matches the supplied documentation.");
-	}
-
-	private void OnLoginSuccess(LoginResult result)
-	{
-		RefreshLeaderboards();
-	}
-
-	private void OnLoginFailed(PlayFabError error)
-	{
-		Debug.LogError("ERROR: Login Failed. Verify Title ID is set correctly.");
-		Debug.LogError(error.GenerateErrorReport());
-	}
-
-	// Selected leaderboard
-	private int CurrentLeaderboard = 0;
-
-	// UX strings for the leaderboards
-	private string[,] LeaderboardInfo = new string[,]
-	{
-		{ "best_time", "Best Time", "Seconds" },
-		{ "high_score", "High Scores", "Points" },
-		{ "total_score", "Total Scores", "Total Points" }
-	};
-
-	public void RefreshLeaderboards()
-	{
-		PlayfabManager.LoadLeaderboards(OnGotLeaderboards, OnGotLeaderboardsFailed);
-	}
-
-	public void GoToLeaderboards()
-	{
-		ShowLeaderboard(CurrentLeaderboard);
-	}
-
-	public void NextLeaderboard()
-	{
-		CurrentLeaderboard++;
-
-		if (CurrentLeaderboard == LeaderboardInfo.GetLength(0))
-		{
-			CurrentLeaderboard = 0;
-		}
-
-		RefreshLeaderboards();
-
-		ShowLeaderboard(CurrentLeaderboard);
```

Comment:
I don't know why the diff tool isn't working on this file. The only change is that I removed this line (ShowLeaderboard(CurrentLeaderboard);)

Reply:
Yikes. Can we open a bug with the GitHub team on this?

Reply:
I'm doing some research. It may be related to a change in line endings. I edited this in Visual Studio and maybe it changed things? I'm investigating before submitting.

Reply:
Yep, it's some sort of whitespace issue. It's easy enough to workaround by telling github to hide whitespace in the diff (the little gear icon right above the diff window). The whitespace for this whole repo is all over the place, so I'm not going to try and correct it now.